src/main.cpp

```diff
@@ -215,24 +236,37 @@ void setup()
   if (servoIndex == -1) {
     Serial.println("Servo could not start!");
   }
-  ESP32_ISR_Servos.setPosition(servoIndex, 0);
+  timeline.add(target)
+    .init(SERVO_MIDPOINT)
+    .offset(1000)
+    .then<Ease::BounceOut>(NO_DANGER, 2000);
+  timeline.start();
+
+  // Force a run 1500ms after startup (to allow servo animation to complete)
```

Comment:
5000-300000 = -295000 - where are you getting 1500ms from?

Reply:
Yeah ok that's my bad. It was 1500ms but I changed it to 5000ms. As to where I'm getting the 5000 from,
`5000ms - 300000ms = -295000ms (-4min,55secs)`
So when the program starts, it sets the last API fetch time to 4 mins and 55 seconds ago. After 5 seconds, the elapsed time ticks over to 5 mins 0 seconds (300000ms), and it refreshes the API.